src/librustc/middle/cstore.rs

```diff
@@ -122,7 +122,6 @@ pub struct ExternCrate {
 /// can be accessed.
 pub trait CrateStore<'tcx> {
     // item info
-    fn describe_def(&self, def: DefId) -> Option<Def>;
```

Comment:
Oh, well. I'll return it back then.
It was completely replaced by `Def`s in `Export`s returned by `item_children`.
`describe_def` in its removed form has a catch though - variant ctors "decay" into variants because they share `DefId`. `item_children` adjust the children lists to correctly fill both namespaces.